server/src/main/java/org/elasticsearch/plugins/ActionPlugin.java

```diff
@@ -48,22 +48,23 @@ public interface ActionPlugin {
     /**
      * Actions added by this plugin.
      */
-    default List<ActionHandler<? extends ActionRequest, ? extends ActionResponse>> getActions() {
+    default Collection<ActionHandler<? extends ActionRequest, ? extends ActionResponse>> getActions() {
```

Comment:
Why are these signatures being changed?

Reply:
I'm normalising them with the other methods in this class, so every method returns `Collection`. They don't need to be ordered, and this is a compatible change due to covariant overrides.